UnitySDK/Assets/ML-Agents/Scripts/CommunicatorObjects/UnityToExternalGrpc.cs

```diff
@@ -2,7 +2,7 @@
 //     Generated by the protocol buffer compiler.  DO NOT EDIT!
 //     source: mlagents/envs/communicator_objects/unity_to_external.proto
 // </auto-generated>
-#pragma warning disable 1591
```

Comment:
Something very fishy is happening right now. On the `develop` branch, there is a line here saying : 

```csharp
# if UNITY_EDITOR || UNITY_STANDALONE_WIN || UNITY_STANDALONE_OSX || UNITY_STANDALONE_LINUX
```

This line is very important and must be included. But I am most worried by the fact that github does not pick it up. There might be other places like this in the PR where code is reverted without github noticing and this is **very** problematic.

Reply:
Update on this, it seems to be because the whole file is conflicting. Maybe this is the reason github does not show the diff here.